prod/openfisca-web-api.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=OpenFisca Web API
```

Comment:
This was on purpose. As @MattiSG [mentioned earlier](https://github.com/openfisca/openfisca-web-api/pull/105#discussion_r107103222), I don't think we should be writing "new" api everywhere.